src/api/app/helpers/webui/request_helper.rb

```diff
@@ -198,4 +198,10 @@ def request_action_header(action, creator)
   def review_request_reason(bs_request, review)
     bs_request.request_history_elements.where(description_extension: review[:id]).pluck(:comment).first.presence || 'No reason given'
   end
+
+  def list_maintainers(maintainers)
+    maintainers.pluck(:login).map do |maintainer|
+      user_with_realname_and_icon(maintainer, no_icon: false, short: true)
```

Comment:
`no_icon: false` isn't needed anymore since it's [the default value](https://github.com/openSUSE/open-build-service/blob/0ae5b0806ecbc1a84f1e222500c26e82a8ee5f1f/src/api/app/helpers/webui/user_helper.rb#L78-L79)

Reply:
I should of noticed that one... Got it corrected!